controllers/humiocluster_defaults.go

```diff
@@ -587,20 +587,23 @@ func (hnp *HumioNodePool) GetContainerReadinessProbe() *corev1.Probe {
 	}
 
 	if hnp.humioNodeSpec.ContainerReadinessProbe == nil {
-		return &corev1.Probe{
+		probe := &corev1.Probe{
 			ProbeHandler: corev1.ProbeHandler{
 				HTTPGet: &corev1.HTTPGetAction{
 					Path:   "/api/v1/is-node-up",
 					Port:   intstr.IntOrString{IntVal: HumioPort},
 					Scheme: hnp.GetProbeScheme(),
 				},
 			},
-			InitialDelaySeconds: 30,
```

Comment:
Curious why we're removing InitialDelaySeconds here?

Reply:
This was an attempt to speed up test execution, but I think I'll revert this specific change for now and keep the `InitialDelaySeconds` as before. Then we can experiment with changes to it after this is merged.

Reply:
When I rebase my changes, I'll revert this.